build.gradle

```diff
@@ -262,6 +263,55 @@ project(":api") {
         withXml = false
     }
 
+    task pom {
+        doLast {
+            conf2ScopeMappings.addMapping(MavenPlugin.COMPILE_PRIORITY + 1, configurations.implementation, 'compile')
+            pom {
+                project {
+                    name "Candlepin API"
+                    description "OpenAPI-based API for Candlepin"
+                    parent {
+                        groupId "org.candlepin"
+                        artifactId "candlepin-parent"
+                        version "2.0"
+                        relativePath ".."
+                    }
+                    packaging "jar"
+                    build([:]) {
+                        plugins {
+                            plugin {
+                                groupId 'org.openapitools'
+                                artifactId 'openapi-generator-maven-plugin'
+                                version '4.0.2'
```

Comment:
Current master is using version 4.2.3

Reply:
Ack, addressed by specifying a var in the buildscript section for openapi_version, and then referencing in buildscript and pom generation section. Also, rebased against master.